src/Uno.UI/Helpers/WinUI/DispatchHelper.cs

```diff
@@ -49,16 +49,17 @@ public DispatcherHelper(DependencyObject dependencyObject = null)
 
 		public void RunAsync(Action func, bool fallbackToThisThread = false)
 		{
-			if (dispatcherQueue != null)
+			// TODO: Uno specific - dispatcher queue is not implemented yet
+			if (false)//dispatcherQueue != null)
```

Comment:
Codacy found an issue: [Remove this useless 'if' statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)